modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase

```diff
@@ -191,6 +191,31 @@ definition {
 		}
 	}
 
+	@description = "LPS-168120. Allow users to rename a custom view"
+	@priority = "4"
+	test CanUpdateName {
+		task ("And Given created custom view") {
+
+			FDSCustomView.createNewView(key_nameView = "TEST");
+
+			Click(locator1 = "FrontendDataSet#SELECT_OPTION_HEADER");
+
+			Click(
+				key_itemName = "Save View",
+				locator1 = "FrontendDataSet#SELECT_OPTION_HEADER");
```

Comment:
These lines are not needed. Only FDSCustomView.createNewView(key_nameView = "TEST"); is needed within the Given task